apps/condo/domains/notification/transports/email.js

```diff
@@ -3,13 +3,60 @@ const FormData = require('form-data')
 
 const conf = require('@core/config')
 
+const { INVITE_NEW_EMPLOYEE_MESSAGE_TYPE } = require('../constants')
+
 const EMAIL_API_CONFIG = (conf.EMAIL_API_CONFIG) ? JSON.parse(conf.EMAIL_API_CONFIG) : null
 
-async function prepareMessageToSend (messageId) {
+async function renderTemplate (message) {
+    // TODO(pahaz): we need to decide where to store templates! HArDCODE!
```

Comment:
For emails, it's good to use HTML templates from services 
Mailgun, Mailchimp, Sendgrid - all have an opportunity to store and edit templates on their side. 
In this case, we can pass only template id and variables for replaces 
By doing so, we can also change templates faster

Reply:
Yes, I needed to make a decision with the product. Hardcoded at the moment